services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -241,7 +241,7 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
     val filePath: String = URLEncoderUtils.encode(path.toString)
     val fileVersion: String = version match {
       case Some(version) => version.name
-      case None          => getDefaultProjectVersion().toString
+      case None          => config.defaultBranch
```

Comment:
Why would we ever want to use branch name as a version?

Reply:
I mean I see the description but it is definetely wrong solution

Reply:
Refactored :)